### ISSUE TITLE
[FEATURE] Notes App, Adapt The Layout To Mobile {branch: dev}.

### ISSUE THREAD

**Issue by ArkadiK94** [thecyberworld/TheCyberHUB#560]
### Detailed description

Adapt The Layout To Mobile
- [ ] when we click on category , all of the inside notes should be displayed only, with back button on top.
- [ ] when open on note, same for that, only content should be displayed with back button.
- [ ] adapt also everything else in notes app to be fit in mobile/tablet.

### Context

It is important so users could use it in mobile and tablet devices 

### Possible implementation

- [ ] when we click on category , all of the inside notes should be displayed only, with back button on top.
- [ ] when open on note, same for that, only content should be displayed with back button.

For the rest , we can discuss the options to implement.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Albert-2** [comment]
hey @ArkadiK94 
i tried to set up the project locally but my window is not allowing to do so cause your project has some malware issue 
can you please take a look into it

**Comment by kabir0x23** [comment]
> hey @ArkadiK94 i tried to set up the project locally but my window is not allowing to do so cause your project has some malware issue can you please take a look into it

hi, yes in project we have malicious data templates, these are just templates, not real malware 
- https://dev.thecyberhub.org/#/tools/revshell (here templates are getting rendered)

pattern is same, that's why some times antivirus got triggered

if u want to work on this project, then u can add this project in exception under antivirus

**Comment by ArkadiK94** [comment]
@Albert-2 hi, how is it going?

**Comment by Albert-2** [comment]
hey @ArkadiK94
sorry brother I'm quite busy this week
hope you will cooperate
Thank you....!! n sorry for inconvenience..

**Comment by E-zech** [comment]
hey i would like to contribute to this project, 
im a junior so i would try to help as much as i know :)

**Comment by ArkadiK94** [comment]
@E-zech Thanks, every help is appreciated so don't worry about it :)
If you have any questions related to this issue or in general, you are welcome to ask :)

**Comment by E-zech** [comment]
actually yes , im trying to registar to the app but i cant , do u have any initial user + pass to login ?

**Comment by E-zech** [comment]
yes i do , the issue is about the goals ? (that they are not responsive)
or is it about the whole dashboard that is not responsive ?
![issue](https://github.com/thecyberworld/TheCyberHUB/assets/135145193/8116440e-4e47-45da-b4f2-0f8679705560)

**Comment by ArkadiK94** [comment]
@E-zech This issue is about Notes feature not Goals feature. Open a different issue for goals but first sync your local repo because this design is old (was changed few days ago).

**Comment by ArkadiK94** [comment]
@E-zech and you are on main environment instead of dev...

**Comment by E-zech** [comment]
im sorry i probably didnt understand the steps of the installation, and jump to the water to soon, i'll learn more about github and then i'll come back, thank u

**Comment by ArkadiK94** [comment]
@E-zech hmm, start from reading the Readme file. Then connect to me if you have any questions. I will keep you assigned to this issue, if you want of course.

**Comment by E-zech** [comment]
will do, for now i think i'll better start with a smaller project.
thank u 4 your time, i'll contact u in the near future :)
